scout/demo/madeline.xml

```diff
@@ -192,7 +192,7 @@ text{
 }
 
 ]]></style>
- <script type="text/javascript" xlink:href="javascript/madeline.js"></script>
+ <script type="text/javascript" xlink:href="/cases/static/madeline.js"></script>
```

Comment:
I just wanted to keep the console error free by default.. and this should be correct currently, though Im not so sure it is what cg does..

Reply:
I'm not sure what that line does 🤔

Reply:
It imports some scripts that the svg uses: init() is the one that gets called and reported missing on the console. We serve it as a static from scout, but I think the path might have changed in cg without that being reflected in scout. Regardless, this works, and removes an error when running demo data. The other part would need some investigation on a live case and an issue to cg.